### ISSUE TITLE
Sample policy resource not getting created

### ISSUE THREAD

**Issue by RinkiyaKeDad** [kubernetes-sigs/wg-policy-prototypes#31]
When following the [Installing guide](https://github.com/kubernetes-sigs/wg-policy-prototypes/tree/master/policy-report#installing) and adding a sample policy resource to the cluster the command fails with the following output:
```
error: unable to recognize "https://github.com/kubernetes-sigs/wg-policy-prototypes/raw/master/policy-report/samples/sample-cis-k8s.yaml": no matches for kind "PolicyReport" in version "policy.kubernetes.io/v1alpha1"
```
If I'm not mistaken then this is because of using an incorrect api version in the [sample resourse file](https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/policy-report/samples/sample-cis-k8s.yaml#L1).
Please let me know if I've got this wrong. I'd like to raise a PR for the same once we can confirm that this is the cause. Thanks :)

**Comment by JimBugwadia** [comment]
Yes, the API version was changed and the sample did not get updated. Thanks for catching this and offering to create a PR!

**Comment by RinkiyaKeDad** [comment]
Fixed by #34
